katcp/resource.py

```diff
@@ -595,10 +595,15 @@ def parent_name(self):
 
     @property
     def name(self):
-        """Name of this KATCP resource"""
+        """Name of this KATCPSensor"""
         return self._name
 
     @property
+    def python_id(self):
```

Comment:
@brickZA: Yeah, I did actually consider that exact name but when talking to @martinslabber I'm realised that the sensor names are also used outside of the context of python (i.e. when a client subscribes to a sensor through katportal)